### PR TITLE
Angular: Convert angular-cli stories to CSF

### PR DESCRIPTION
**Pull request by shilman**
Issue: #7101 

## What I did

Ran `sb migrate storieof-to-csf` and minor touchup

## How to test

```
cd examples/angular-cli
yarn storybook
```

### CODE REVIEW

File: examples/angular-cli/src/stories/addon-actions.stories.ts [hunk 1]
```diff
@@ -1,23 +1,34 @@
...
+
+export const actionOnly = () => ({
+  component: Button,
+  props: {
+    text: 'Action only',
+    onClick: action('log 1'),
+  },
+});
+
+actionOnly.story = {
```
Comment (kroeder) [inline examples/angular-cli/src/stories/addon-actions.stories.ts]: Is there a way to type this? In `strict: true` you most likely have to 
`export const actionOnly: any` in order to call `.story`

Maybe `actionOnly: Story = () => { ... }`?

Comment (shilman) [reply examples/angular-cli/src/stories/addon-actions.stories.ts]: @kroeder great idea!

https://github.com/storybookjs/storybook/issues/7677

**Comment by shilman** [comment]
@igor-dv angularshots seems to be failing on this PR and i can repro on my dev machine. any chance you can take a look for me? i think it's angular-specific since `official-storybook` is also using CSF & typescript successfully.

**Comment by shilman** [comment]
Self-merging

**Pull request merged**